### ISSUE TITLE
Suggestion: General UI Readability Improvements

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#339]
**Is your feature request related to a problem? Please describe.**
There are some significant shortcomings in terms of readability in certain parts of the app that manifest at least in windows (and I presume in other os's since it relates to the pyside stylesheet mostly). 

- [ ] 1. Radio button text is currently dark on dark.  (No styling exists for radio buttons currently)
- [ ] 2. Some items such as buttons have padding issues where the text is very close to the left and right edges (There's only 1px of padding around buttons. It really should be more. As an example, material design common buttons have left right padding of 24px, though even a padding of just 5px would be better.)
- [ ] 3. The menu bar of the app in terms of styling is rather inconsistent with the rest of the app on windows. I don't think any styling exists for it. My understanding is that it defaults to system default for mac/linux, but not on windows which is rather ugly without styling.

**Describe the solution you'd like**
I'm willing to help make these changes in a pull request, I'm just opening this issue first for purposes of discussion.
